### ISSUE TITLE
go-fed.org is down

### ISSUE THREAD

**Issue by anderspitman** [go-fed/activity#161]

**Comment by gabek** [comment]
I don't think this should be a surprise to anybody. It was bound to happen, as there's no longer a maintainer.

**Comment by anderspitman** [comment]
I'm new to the community. Is anyone in contact with the repo owner?

**Comment by tsmethurst** [comment]
> Is anyone in contact with the repo owner?

As far as I know, no, I think CJ has moved on to other things :)
